.build-tools/builtin-authentication-profiles.yaml

```diff
@@ -103,3 +103,70 @@ azuread:
           - AzurePublicCloud
           - AzureChinaCloud
           - AzureUSGovernmentCloud
+
+gcp:
+  - title: "GCP API call Authentication"
+    description: |
+      Authenticate authenticates API calls with the given service account or refresh token JSON credentials.
+    metadata:
+      - name: private_key_id
+        required: true
+        sensitive: true
+        description: |
+          GCP private key id.
+        example: '"privateKeyId"'
+      - name: private_key
+        required: true
+        sensitive: true
+        description: |
+          GCP credentials private key. Replace with x509 cert.
+        example: '"-----BEGIN PRIVATE KEY-----\nMIIEvgIBADAsUw+2/y/MfhhkIKQTgnoiIZFHximeQN67VF9pifNR/D0k\\n-----END PRIVATE KEY-----\n"'
+      - name: type
```

Comment:
Same feedback: the description says "private key", but then you mention a X.509 certificate. Is it a certificate or key?